inspirehep/modules/workflows/workflows/article.py

```diff
@@ -120,6 +120,20 @@
     ),
 ]
 
+CHECK_AUTO_ACCEPT = [
```

Comment:
Dyslexic me, it's approved, not accept :/

Reply:
ok ok on it :+1: